Packs/MISP/Integrations/MISPV3/MISPV3.py

```diff
@@ -807,11 +828,57 @@ def get_indicator_results(
                               outputs_prefix='MISP.Attribute',
                               outputs_key_field='ID',
                               readable_output=readable_output)
+
     else:
+        if search_warninglists:
+            values = argToList(value)
+            res: list = []
+            human_readable: str = ""
+            misp_warninglists_response = PYMISP.values_in_warninglist(values)
+            if 'errors' in misp_warninglists_response:
+                raise DemistoException(
+                    f'Unable to validate against MISP warninglists!\nError message: {misp_warninglists_response}')
+            if (misp_warninglists_response and isinstance(misp_warninglists_response, dict)
+                    and len(misp_warninglists_response.items()) > 0):
+                list_key: str = list(misp_warninglists_response.keys())[0]
```

Comment:
instead of lines 843+844 we can do lists = list(misp_warninglists_response.values())[0]